{"name": "test_add_new_data", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\TT\\\\Desktop\\\\newStudent.json'", "trace": ">   ???\nE   FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\TT\\\\Desktop\\\\newStudent.json'\n\nC:\\Users\\TT\\PycharmProjects\\REST Automation 1\\TestCases\\test_end_to_end.py:8: FileNotFoundError"}, "start": 1608055588572, "stop": 1608055588572, "uuid": "46629a70-b07d-42c4-86db-419c7354b9f0", "historyId": "9195310cc686a68896307ece1f2b8b46", "testCaseId": "60dc01ab92e369a1a36a634d515e0524", "fullName": "TestCases.test_end_to_end#test_add_new_data", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_end_to_end"}, {"name": "host", "value": "DESKTOP-NMI0QKM"}, {"name": "thread", "value": "6852-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_end_to_end"}]}